packages/relay/src/lib/clients/cache/redisCache.ts

```diff
@@ -249,10 +249,10 @@ export class RedisCache implements IRedisCacheClient {
    * @returns {Promise<void>} A Promise that resolves when the client is disconnected.
    */
   async disconnect(): Promise<void> {
-    await this.getConnectedClient().then((client) => {
-      client.disconnect();
-      client.unsubscribe();
-    });
+    if (await this.connected) {
+      const client = await this.getConnectedClient();
+      await client.disconnect();
+    }
```

Comment:
Is there a reason we are using `client.disconnect()` instead of `client.quit()` ?  I thought `quit` was cleaner.  It waits for pending replies to be received before closing.

Reply:
Thanks for pointing this out! I changed it to use `quit()`.